user_manual/ios_app.rst

```diff
@@ -72,7 +72,7 @@ Sharing Files
 ~~~~~~~~~~~~~
 
 The little arrow button on the bottom left has options for sharing the file via
-email and instant messaging, saving or copying, and printing. 
+email, instant messaging or any other of your apps, saving or copying. 
```

Comment:
This segment of the sentence is slightly confusing. I suggest writing it as "email, instant messaging (or any other of your apps), as well as saving and copying."